packages/webapp/src/hooks/useLogs.tsx

```diff
@@ -39,3 +41,54 @@ export function useSearchLogs(env: string, body: SearchLogs['Body']) {
 
     return { data, error, loading };
 }
+
+export function useGetOperation(env: string, params: GetOperation['Params']) {
+    const { data, error } = useSWR<GetOperation['Success']>(`/api/v1/logs/operations/${params.operationId}?env=${env}`, swrFetcher);
+
+    const loading = !data && !error;
+
+    return {
+        loading,
+        error,
+        operation: data?.data
+    };
+}
+
+export function useSearchMessages(env: string, body: SearchMessages['Body']) {
+    const [loading, setLoading] = useState<boolean>(false);
+    const [data, setData] = useState<SearchMessages['Success']>();
+    const [error, setError] = useState<SearchMessages['Errors']>();
+
+    async function fetchData() {
+        setLoading(true);
+        try {
+            const res = await fetch(`/api/v1/logs/messages?env=${env}`, {
+                method: 'POST',
+                body: JSON.stringify(body),
+                headers: { 'Content-Type': 'application/json' }
+            });
+            if (res.status !== 200) {
+                setData(undefined);
+                setError((await res.json()) as SearchMessages['Errors']);
+                return;
+            }
+
+            setError(undefined);
+            setData((await res.json()) as SearchMessages['Success']);
+        } catch (err) {
+            console.log(err);
```

Comment:
If the log should stay then at least it should console.error

Reply:
it should not, removing 👍🏻